build.gradle

```diff
@@ -152,6 +146,9 @@ configure(javaMainProjects) {
         compileOnly 'com.google.code.findbugs:jsr305:1.3.9'
         api 'org.slf4j:slf4j-api:1.7.6', optional
         testImplementation 'com.google.code.findbugs:jsr305:1.3.9'
+
+        // https://issues.apache.org/jira/browse/GROOVY-10194
+        testImplementation 'org.codehaus.groovy:groovy-all:3.0.9'
```

Comment:
Force the latest version of groovy - allows the Spock tests to compile and then run! (was a pain to find and fix this).

Reply:
Oh, just needed the latest patch release.  So once Spock re-releases we can remove this?

Reply:
Should we have a ticket to update Spock dependency, and mention this?

Reply:
Can do - https://repo1.maven.org/maven2/org/spockframework/spock-core/2.0-groovy-3.0/spock-core-2.0-groovy-3.0.pom requires 3.0.8 :/

Spock 2.1 will require Groovy 3.0.9 but isn't released yet.
Spock 2.2 will possibly add a Groovy  4.0 release